docs/fidesops/docs/quickstart.md

```diff
@@ -0,0 +1,89 @@
+# Quickstart
+The [Fidesops repository](https://github.com/ethyca/fidesops) includes a built-in docker compose configuration for quickly experimenting with a working demo environment. 
```

Comment:
I was a little confused by this ticket because I thought the "autopilot" quickstart would be referenced here https://github.com/ethyca/fidesops#rocket-quick-start.

If we have this Quickstart, and then a different quickstart section in the landing page of the fidesops repo, should we rename one of them to not cause confusion?

Reply:
I'm down to rename this one in particular (and maybe link to the auto quickstart in the opening here). I wanted to address the [deployment guide](https://ethyca.github.io/fidesops/deployment/) mentioning there's some built-in configuration, but no real documentation on how to spin things up to poke around. 

Alternatively... the readme could make sense as more of a "test drive," whereas this follows the pattern set by Fidesctl for a local Quick Start?  🤔

Reply:
either one is fine with me, I just didn't want two different quickstarts!